lib/KeyboardAwareHOC.js

```diff
@@ -609,7 +609,7 @@ function KeyboardAwareHOC(
     }
 
     _extraScrollHeight() {
-      return this.props.extraScrollHeight + this._totalVerticalDistanceToWindow();
+      return this.props.extraScrollHeight + this.topDistanceToWindow;
```

Comment:
I noticed that adding the extra bottom height was resulting in wrong calculations. Using the top extra height works well for both iPhones and iPads.

Reply:
An update here:
Together with the keyboard height, the bottom SafeArea height is already included, that's why adding it again will lead to an incorrect measurement.

On iPads there is no bottom SafeArea, and there's still a keyboard-related bar that gives the proper measurement, but on iPhones with external keyboard, it IS necessary to manually add the bottom SafeArea. So I will try to find the solution for that now.